networkx/generators/tests/test_expanders.py

```diff
@@ -70,3 +70,18 @@ def test_margulis_gabber_galil_graph_badinput(graph_type):
         nx.NetworkXError, match="`create_using` must be an undirected multigraph"
     ):
         nx.margulis_gabber_galil_graph(3, create_using=graph_type)
+
+
+@pytest.mark.parametrize("graph_type", (nx.Graph, nx.DiGraph))
+def test_chordal_cycle_graph_badinput(graph_type):
```

Comment:
This all looks good! Thanks @chimaobi-okite!

could you also add `nx.MultiDiGraph` ?

Reply:
Yes I can, I will do that now. Thank you!